notifier/notifier_test.go

```diff
@@ -117,6 +117,27 @@ func TestFailSendEvent(t *testing.T) {
 	time.Sleep(time.Second * 2)
 }
 
+func TestNoResendForSendToBrokenContact(t *testing.T) {
```

Comment:
Let's assume this test as a slow test and will skip it on short testing
```go
    if testing.Short() {
        t.Skip("skipping test in short mode.")
    }
```

Reply:
fixed